src/components/_overlays/ContextMenu/ContextMenu.styles.ts

```diff
@@ -1,27 +1,15 @@
 import styled from '@emotion/styled'
 
 import { Text } from '@/components/Text'
-import { cVar, oldColors, sizes, transitions } from '@/styles'
+import { cVar, sizes, transitions } from '@/styles'
 
 export const StyledContainer = styled.div`
-  background-color: ${oldColors.gray[800]};
+  background-color: ${cVar('colorBackgroundStrong')};
   width: 200px;
-  color: ${oldColors.white};
+  color: ${cVar('colorText')};
```

Comment:
I don't think we need this style here, the text color will come from the list item component